kotsadm/pkg/redact/redact.go

```diff
@@ -186,6 +186,52 @@ func SetRedactYaml(slug, description string, enabled, newRedact bool, yamlBytes
 	return redactorEntry, nil
 }
 
+// sets whether an individual redactor is enabled
+func SetRedactEnabled(slug string, enabled bool) (*RedactorMetadata, error) {
+	configMap, _, err := getConfigmap()
+	if err != nil {
+		return nil, err
+	}
+
+	newData, redactorEntry, err := setRedactEnabled(slug, enabled, time.Now(), configMap.Data)
+	if err != nil {
+		return nil, err
```

Comment:
dito

Reply:
this one shouldn't be errors.wrap, since it's an internal function only called in one place
the functions are separated for testing purposes